src/sdsc/__init__.py

```diff
@@ -965,20 +966,18 @@ def initialize():
     if sdsc_initialized:
         return True
 
-    # Prepare parser (add py: namespace)
-    ns = etree.FunctionNamespace('https://www.github.com/openSUSE/suse-doc-style-checker')
-    ns.prefix = 'py'
-    ns.update(dict(linenumber=linenumber, termcheck=termcheck,
-                   buildtermdata=buildtermdata, dupecheck=dupecheck,
-                   sentencelengthcheck=sentencelengthcheck,
-                   sentencesegmenter=sentencesegmenter,
-                   tokenizer=tokenizer, counttokens=counttokens,
-                   splitpath=splitpath))
-
-    global parser
-    parser = etree.XMLParser(ns_clean=True,
-                             remove_pis=False,
-                             dtd_validation=False)
+    # Update namespace dictionary with XSLT extension functions
+    # TODO: Move this into a separate file
+    SDSCNS.update(dict(linenumber=linenumber,
```

Comment:
Could that be just "namespace"? Even just seeing that is painful, typing it probably even more so.